portfolio/src/main/java/com/google/sps/data/CommentSection.java

```diff
@@ -26,14 +31,30 @@ public CommentSection() {
     queryLimit = initialQueryLimit;
   }
 
-  public void addComment(String title, String content, long timestamp) {
+  public void addComment(String title, String content, long timestamp) throws IOException {
     Entity commentEntity = new Entity("Comment");
     commentEntity.setProperty("title", title);
     commentEntity.setProperty("content", content);
     commentEntity.setProperty("timestamp", timestamp);
+    commentEntity.setProperty("sentiment", analyzeSentiment(content));
     datastore.put(commentEntity);
   }
 
+  public String analyzeSentiment(String text) throws IOException {
+    Document doc = Document.newBuilder().setContent(text).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    languageService.close();
+    if (score < -0.33) {
```

Comment:
where does -0.33 come from? It's better to put it as a const outside this function (similar to initialQueryLimit).